src/geometries/CapsuleGeometry.js

```diff
@@ -0,0 +1,49 @@
+import { Path } from '../extras/core/Path';
+import { LatheGeometry } from './LatheGeometry';
+
+class CapsuleGeometry extends LatheGeometry {
+
+	/**
+	 * Creates a CapsuleGeometry.
+	 *
+	 * @param {number} radiusTop — Radius of the emisphere at the top. Optional; defaults to 1.
+	 * @param {number} radiusBottom — Radius of the emisphere at the bottom. Optional; defaults to 1.
+	 * @param {number} height — Height of the middle section. Optional; defaults to 1.
+	 * @param {number} capSegments — Number of curve segments used to build the caps. Optional; defaults to 4.
+	 * @param {number} radialSegments — Number of segmented faces around the circumference of the capsule. Optional; defaults to 8.
+	 */
+	constructor( radiusTop = 1, radiusBottom = 1, height = 1, capSegments = 4, radialSegments = 8 ) {
+
+		const R1 = radiusTop - radiusBottom;
+		// max here prevents failing when height is 0
+		const a = Math.asin( R1 / Math.max( Number.EPSILON, height ) );
```

Comment:
I added this check to avoid a void geometry. 
Problems still arise when passing certain combinations of radii and height, should we add more checks/failsafes?